pkg/server/router/schema.go

```diff
@@ -39,20 +39,23 @@ type CreateSchemaRequest struct {
 	// The schema must be against draft 2020-12, 2019-09, or 7.
 	Schema schemalib.JSONSchema `json:"schema" validate:"required"`
 
-	// TODO(gabe): re-enable in https://github.com/TBD54566975/ssi-service/issues/493
-	// Sign represents whether the schema should be signed by the author. Default is false.
-	// If sign is true, the schema will be signed by the issuer's private key with the specified KID
-	// Sign bool `json:"sign,omitempty"`
-	// Issuer represents the DID of the issuer for the schema if it's signed. Required if sign is true.
-	// Issuer string `json:"issuer,omitempty"`
-	// IssuerKID represents the KID of the issuer's private key to sign the schema. Required if sign is true.
-	// IssuerKID string `json:"issuerKid,omitempty"`
+	// Issuer represents the DID of the issuer for the schema if it's signed. Required if intending to sign the
+	// schema as a credential using CredentialSchema2023.
+	Issuer string `json:"issuer,omitempty"`
+	// IssuerKID represents the KID of the issuer's private key to sign the schema. Required if intending to sign the
+	// schema as a credential using CredentialSchema2023.
+	IssuerKID string `json:"issuerKid,omitempty"`
```

Comment:
I suggest moving these two fields into a separate struct. The presence of the struct would then signal the intention of the user. When present, then both fields should be populated, and the intention of the user is clear that they want a CredentialSchema2023. When absent, then the user wants to do a JsonSchema2023.

Reply:
good idea